web_custom_login/__init__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```

Comment:
In particular, init.py files usually dont have copyright notice. for any other py file please use simplified copyright notice.